package.json

```diff
@@ -74,11 +72,15 @@
     "ember-route-action-helper": "^1.0.0",
     "ember-source": "~2.11.0",
     "ember-test-helpers": "~0.6.3",
+    "eslint-plugin-ember": "^5.0.0",
```

Comment:
Please use a ~ here. New minor versions of `eslint-plugin-ember` tend to introduce new rules which cause builds to fail.